app/views/categories/index.html.erb

```diff
@@ -1,4 +1,11 @@
-<h1>Categories</h1>
+<h1>
+    Categories 
+    <span class="btn-group float-right">
+        <%= link_to "All", categories_path, class: 'btn btn-default' %>
+        <%= link_to "Learned", categories_path(filter: "learned"), class: 'btn btn-default' %>
```

Comment:
@CharlotteDS 
Highlight the current selected state, so user can know which status is being shown, like below:
![image](https://user-images.githubusercontent.com/20507565/50734425-f3f4e880-11d9-11e9-8fb7-2c0b4fcbfeb9.png)